officers/deanna_troi.json

```diff
@@ -0,0 +1,89 @@
+{
+  "description": "Deanna Troi is the half-human, half-Betazoid counselor aboard the USS Enterprise. Her passion for the emotional wellbeing of her fellow crewmates is only rivalled by her love of chocolate. As a member of the senior staff, and a regular presence on the bridge, Troi has become an invaluable advisor to Captain Picard in times of great strife. Her expertise has placed her front and center during many first contact situations, with her empatic abilities - a key tool in sensing vital information at critical times. Troi is a beloved member of the Enterprise crew, and in her downtime enjoys socializing with her friends in Ten Forward, exercising, or joining the weekly poker night. Deanna has a close relationship with her mother, Lwaxanna Troi, who frequently visits the Enterprise. Deanna is also quick to action and decisive when needs arise, and is a fully trained command officer.",
+  "ability_info": {
+    "captain_maneuver": {
+      "name": "Telepathic Predictions",
+      "bonus": "Deanna Troi reduces the Crit Chance of Armada Targets by {value}",
+      "percentage": true,
+      "value": 0.15
+    },
+    "officer_ability": {
+      "name": "Nemesis",
+      "bonus": "Deanna Troi increases Weapon Damage against Romulan Hostiles and Armadas by {value}",
+      "percentage": true
+    }
+  },
+  "class": "science",
+  "faction": "federation",
+  "group": "tng crew",
+  "img_url": "./assets/deanna_troi.png",
+  "officer_name": "Deanna Troi",
+  "ranks": [
+    {
+      "name": "Ensign I",
+      "officer_ability_value": 1,
+      "rank_id": 1,
+      "requirements": {
+        "badges": 0,
+        "credits": 0,
+        "others": [],
+        "shards": 100,
+        "xp": 0
+      }
+    },
+    {
+      "name": "Lieutenant JG II",
+      "officer_ability_value": 1.5,
+      "rank_id": 2,
+      "requirements": {
+        "badges": 0,
+        "credits": 1100,
+        "others": [],
+        "shards": 100,
+        "xp": 50000
+      }
+    },
+    {
+      "name": "Lieutenant III",
+      "officer_ability_value": 2,
+      "rank_id": 3,
+      "requirements": {
+        "badges": 1,
+        "credits": 4400,
+        "others": [],
+        "shards": 200,
+        "xp": 500000
+      }
+    },
+    {
+      "name": "Lt. Commander IV",
+      "officer_ability_value": 2.5,
+      "rank_id": 4,
+      "requirements": {
+        "badges": 2,
+        "credits": 11500,
+        "others": [],
+        "shards": 300,
+        "xp": 1250000
+      }
+    },
+    {
+      "name": "Commander V",
+      "officer_ability_value": 3,
+      "rank_id": 5,
+      "requirements": {
+        "badges": 4,
+        "credits": 23000,
+        "others": [],
+        "shards": 800,
+        "xp": 2500000
+      }
+    }
+  ],
+  "rarity": "epic",
+  "synergy": {
```

Comment:
I think those might be wrong. I think it's 0.02 for science and 0.05 for engineering and command.